apps/meteor/ee/app/license/server/license.ts

```diff
@@ -29,6 +34,26 @@ class LicenseClass {
 
 	private modules = new Set<string>();
 
+	private appsConfig: NonNullable<ILicense['apps']> = {
+		maxPrivateApps: 3,
+		maxMarketplaceApps: 5,
+	};
+
+	private Apps: AppServerOrchestrator;
+
+	constructor() {
+		/**
+		 * Importing the Apps variable statically at the top of the file causes a change
+		 * in the import order and ends up causing an error during the server initialization
+		 *
+		 * We added a dynamic import here to avoid this issue
+		 */
+		// eslint-disable-next-line @typescript-eslint/naming-convention
+		import('../../../../app/apps/server').then(({ Apps }) => {
+			this.Apps = Apps;
+		});
```

Comment:
This will end up being removed in favor of the new Apps Service as soon as the microservice migration is merged